NT sponsored, experimental botany shuttle

Ultravogan opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
adds a botany admeme shuttle
5 CentComm botanist have been sent to the station to sell plants they have experimented on and document how the crew reacts to the plants when eaten, used, or in the case of the cannabis an tobacco, smoked

## Why we need to add this
for the funnies and for a botany admeme

## Media (Video/Screenshots)
no changes, just adds an admeme event with no extra mechanics

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
added a botany admeme shuttle

:cl: vlorgan
- add: Admeme botany shuttle

Ultravogan committed:
Add files via upload

an admeme shuttle where NT helped a central command botanist named "johhny planter" create a shuttle to test and document the effects of a chemical mix known as "evil plant mix" that causes extreme mutations when used on freshly planted seeds. in exchange for the shuttle, NT wanted him and some other botanist to test and document the effects it has on the plants grown with the chemical mix and used on the crew of stations, especially when used on cannabis seeds.

Ultravogan committed:
botany admeme shuttle 

an admeme shuttle where NT helped a central command botanist named "johhny planter" create a shuttle to test and document the effects of a chemical mix known as "evil plant mix" that causes extreme mutations when used on freshly planted seeds. in exchange for the shuttle, NT wanted him and some other botanist to test and document the effects it has on the plants grown with the chemical mix and used on the crew of stations, especially when used on cannabis seeds.

ScorchFollower commented:
Yup, looks good. Do be sure to change the name and specific what you are wanting a bit more and why. Do be sure to get that title renamed.

ScorchFollower commented:
Yup, good work!

PubliclyExecutedPig commented:
Can you provide a screenshot pls?

Ultravogan committed:
Update events.yml

adds a prototype to my experimental botany shuttle

Ultravogan commented:
> Can you provide a screenshot pls?

![botany shuttle 1](https://github.com/user-attachments/assets/d74b1130-2302-4702-9a7c-28f0d08365be)
![botany shuttle 2](https://github.com/user-attachments/assets/818d1de0-ed3c-4756-b3da-3a53d0fbf515)
![botany shuttle 3](https://github.com/user-attachments/assets/e2045740-8a53-4fc6-8aac-6f5242743393)
 there ya go?
 also added the prototype for it

PubliclyExecutedPig reviewed (approved):
Looks good

Happyrobot33 commented:
Malformed changelog, please look at other PRs to understand the proper way of documenting changes for end users

Ultravogan commented:
i am new to this. exactly what part of the changelog was wrong.....

Happyrobot33 commented on Resources/Prototypes/_StarLight/GameRules/events.yml:
```diff
@@ -155,3 +155,13 @@
+  components:
+    - type: RuleGrids
+    - type: LoadMapRule
+    - gridPath: /Maps/Shuttles/NT-Experimental-Botany-shuttle.yml
```
Removing the - here should fix this. Please test your changes properly in the future

Ultravogan replied:
fixed it, but i had it like that cause that was the filename for the shuttle....

Happyrobot33 replied:
Not that -...... The one at the start of the line needs to be removed

Ultravogan replied:
![image](https://github.com/user-attachments/assets/fa4eaeeb-6443-4d00-b609-dba93e7bbe23)

Happyrobot33 replied:
You need to remove the dash at the start of this line. It should be gridPath not - gridpath

The - in the path was fine

ALSO the shuttle file should be in the starlight folder for shuttles, not the wizden folder for shuttles. Its in Maps/_Starlight/Shuttles if i recall properly

Ultravogan committed:
Update events.yml

fixed code

Ultravogan committed:
Update events.yml

refixed.... i hope

Ultravogan committed:
Delete Resources/Maps/Shuttles/NT-Experimental-Botany-shuttle.yml

removed to move to another folder

Ultravogan committed:
Update events.yml

Ultravogan committed:
admeme experimental botany shuttle

an admeme shuttle where NT helped a central command botanist named "johhny planter" create a shuttle to test and document the effects of a chemical mix known as "evil plant mix" that causes extreme mutations when used on freshly planted seeds. in exchange for the shuttle, NT wanted him and some other botanist to test and document the effects it has on the plants grown with the chemical mix and used on the crew of stations, especially when used on cannabis seeds.

Ultravogan commented:
ok. i THINK it's all fixed now.

Happyrobot33 committed:
Rename the shuttle
